static/app/plugins/components/pluginIcon.tsx

```diff
@@ -149,23 +147,12 @@ const FallbackPluginIcon = styled('div')<Props>`
     (pluginId !== undefined && ICON_PATHS[pluginId]) || DEFAULT_ICON});
 `;
 
-const PluginIcon = ({pluginId, size, sentryApp, isColor, className}: Props) => (
-  <Feature features={['organizations:sentry-app-logo-upload']}>
-    {({hasFeature}) => {
-      if (hasFeature && sentryApp) {
-        return (
-          <Avatar
-            size={size}
-            sentryApp={sentryApp}
-            isColor={isColor}
-            className={className}
-          />
-        );
-      }
-      return <FallbackPluginIcon pluginId={pluginId} size={size} className={className} />;
-    }}
-  </Feature>
-);
+const PluginIcon = ({pluginId, size, isAvatar = false, avatarProps, className}: Props) =>
```

Comment:
@leeandher Is there a way we can restore the PluginIcon component to how it was originally? https://github.com/getsentry/sentry/blob/1f0596677fa3a4d97056ccb77144ca8895cf015a/static/app/plugins/components/pluginIcon.tsx

Can we handle the logic if we want an avatar at a higher level?

Reply:
Restored it, The check for the feature flag happens in SentryAppIcon now